### PR TITLE
Project Landing Page: Add missing word

### PR DESCRIPTION
**Pull request by oliverjhn**
## Because
A word was missing

## This PR

- Adds the word 'In' as it was missing from context

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/html_css/flexbox/project-landing-page.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ For this project you'll be creating an entire web page from a design we'll provi
 
 The design we're providing you comes in the form of 2 images: one is an image of the complete website, and one has some details about some of the fonts and colors we've used.
 
-Do _not_ be afraid to use google or go back to previous lessons to look something up. **Real-life, professional developers use google _constantly_ for things that they have been doing for years.** At this point it is not expected that you will have everything memorized, so don't worry about it. Additionally, there are a few small details that you may not have encountered in our lessons yet. _This is by design._ These details are minor, and easily searched (e.g. google `css rounded corners`).
+Do _not_ be afraid to use google or go back to previous lessons to look something up. **In Real-life, professional developers use google _constantly_ for things that they have been doing for years.** At this point it is not expected that you will have everything memorized, so don't worry about it. Additionally, there are a few small details that you may not have encountered in our lessons yet. _This is by design._ These details are minor, and easily searched (e.g. google `css rounded corners`).
```
Comment (rlmoser99) [inline foundations/html_css/flexbox/project-landing-page.md:7]: In this example, "Real-life, professional developers", "Real-life" is an adjective. Here is another example of this use: "real-life heroes that changed the world...".

If you want to make this change, "Real-life" should be changed to "In real life" without the dash.

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
Thank you for taking the time to make this PR. There is one small change that is needed before I can approve and merge.

File: foundations/html_css/flexbox/project-landing-page.md (hunk 1 above)
Comment (oliverjhn) [reply foundations/html_css/flexbox/project-landing-page.md]: Thank you for your feedback. I fixed the error.

**Review by rlmoser99 — APPROVED** [review]
Thanks for making these changes so quickly! 🚀

**Pull request merged**